core/src/main/scala/org/http4s/syntax/AllSyntax.scala

```diff
@@ -4,8 +4,6 @@ package syntax
 trait AllSyntax
     extends AnyRef
     with AsyncSyntax
-    with EffectResponseSyntax
-    with EffectRequestSyntax
```

Comment:
Getting rid of these is going to be rough on migrants from < 0.18.  Would be nice if these still compiled.  Not mandatory, but worth an effort.

Reply:
The problem is implicit on implicit here. There is no core set of traits to expand anymore. It could be manageable by creating any EffectMessage Syntax. But in the process these all became concrete rather than traits. We have a `Request[F]` and `Response[F]`, that happen to both satisfy `Message[M[_[_]], F[_]]`.